web-components/src/components/inputs/NumberTextField.tsx

```diff
@@ -21,7 +21,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     position: 'absolute',
     right: 0,
     top: 0,
-    height: '100%',
+    height: 60,
```

Comment:
Match input height.
Before we had this:
![image](https://user-images.githubusercontent.com/530644/187912644-197260d9-0f05-4d0a-9450-aab94908c6b4.png)